lib/src/log.rs

```diff
@@ -285,7 +291,7 @@ pub struct DAGSetSchedulerLog {
     dag_set_info: DAGSetInfo,
     processor_info: ProcessorInfo,
     dag_set_log: Vec<DAGLog>,
-    node_set_logs: Vec<Vec<NodeLog>>,
+    node_set_logs: Vec<Vec<JobLog>>, // node_set_logs[dag_id][job_index]
```

Comment:
This comment is not valid with preemption.

Reply:
Deleted.